plugins/formal-verification/testData/diagnostics/bad_contracts/binary_search.kt

```diff
@@ -0,0 +1,54 @@
+// USE_STDLIB
+
+import org.jetbrains.kotlin.formver.plugin.AlwaysVerify
+
+@AlwaysVerify
+fun <!VIPER_TEXT!>mid_increased_by_one<!>(arr: List<Int>, target: Int): Boolean {
+    val size = arr.size
```

Comment:
Out of curiosity: is putting the size in a local like this necessary, or is it just for convenience?

Reply:
just for convenience, it works also without these local vars